source/help/getting-started/organizing-conversations.rst

```diff
@@ -8,17 +8,19 @@ Channels are used to organize conversations across different topics. There are t
 
 Public Channels
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Public Channels are open to everyone on a team. New team members are automatically a part of two Public Channels: Town Square and Off-Topic.  
+Public Channels are open to everyone on a team. New team members are automatically a part of two Public Channels: Town Square and Off-Topic.
```

Comment:
"...are automatically a part of..." > "are automatically added to two Public Channels upon sign-up to the team:..."